File: joseffe-barroso/Python/aula04/while/exercise40.py
index = 1

firstNumber = -1
secondNumber = 1
thirdNumber = 1
sumNumbers = 0

while (index < 31):
    print(secondNumber)
    sumNumbers = secondNumber + firstNumber + thirdNumber
    firstNumber = secondNumber
    secondNumber = thirdNumber
    thirdNumber = sumNumbers
    index = index + 1

File: joseffe-barroso/Java/src/aula04/While/Exercise40.java
package While;

public class Exercise40 {

	public static void main(String[] args) {
		int index = 1, firstNumber = -1, secondNumber = 1, thirdNumber = 1, sumNumbers = 0;

		while (index < 31) {
			System.out.printf("\n%d", secondNumber);
			sumNumbers = secondNumber + firstNumber + thirdNumber;
			firstNumber = secondNumber;
			secondNumber = thirdNumber;
			thirdNumber = sumNumbers;
			index = index + 1;
		}
	}

}